tasks/oah-validate.yml

```diff
@@ -0,0 +1,25 @@
+---
+
+- name: Ensure Docker is installed and command is running
+  command: docker --version
+  register: docker_command
+  ignore_errors: true
+
+- name: Docker is installed
+  debug:
+    msg: "{% if docker_command.rc == 0 %} Docker is installed {% else %} Docker Not found {% endif %}"
+
+- name: Ensure /etc/docker/ directory exists.
+  file:
+    path: /etc/docker
+    state: directory
+    mode: 0755
+  when: docker_daemon_options.keys() | length > 0
+
+- name: Ensure Docker is started and enabled at boot.
```

Comment:
Failed to start service after removing docker.

![image](https://user-images.githubusercontent.com/58501088/190403905-67b86c2b-6c59-49c2-9399-b40d15b52cbf.png)

Reply:
Removed the boot up service check task. As we don't need docker to run at startup and don't need to validate the service. 

If docker not installed , message is being displayed and same for not installed. PFA
![image](https://user-images.githubusercontent.com/36243506/190912730-e64d6639-e553-4132-9d61-82062a211562.png)